api/nodes/serializers.py

```diff
@@ -75,6 +77,9 @@ class NodeSerializer(JSONAPISerializer):
     files = JSONAPIHyperlinkedIdentityField(view_name='nodes:node-providers', lookup_field='pk', lookup_url_kwarg='node_id',
                                              link_type='related')
 
+    comments = JSONAPIHyperlinkedIdentityField(view_name='nodes:node-comments', lookup_field='pk', lookup_url_kwarg='node_id',
+                                               link_type='related', meta={'unread_comments_count': 'get_unread_comments_count'})
```

Comment:
Yeah, I followed how the API was handling counts for other node properties. Do you have any suggestions for alternatives to hard-coding the key names?